permissions/plugin-custom-folder-icon.yml

```
@@ -0,0 +1,6 @@
+---
+name: "custom-folder-icon"
+github: "jenkinsci/custom-folder-icon-plugin"
+paths:
+- "org/jenkins-ci/plugins/custom-folder-icon"
+developers: []
```

you might want to add yourself here?

```suggestion
developers:
- strangelookingnerd
```